pkg/features/kube_features.go

```diff
@@ -1270,6 +1270,10 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.StorageVersionHash: {Default: true, PreRelease: featuregate.Beta},
 
+	genericfeatures.StrictCostEnforcementForVAP: {Default: false, PreRelease: featuregate.Alpha}, // deprecating in 1.32
+
+	genericfeatures.StrictCostEnforcementForWebhooks: {Default: false, PreRelease: featuregate.Alpha}, // deprecating in 1.32
```

Comment:
I agree with default-off until we have a better per-object detection / mitigation story

In terms of stability level, I think these are at least beta+

(have to adjust in genericfeatures file as well, I guess)

Reply:
Updated to beta for both FG